schemas/com.snowplowanalytics.telemetry/iglu_repo_telemetry/jsonschema/1-0-0

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for telemetry of an Iglu repository",
+  "self": {
+    "vendor": "com.snowplowanalytics.telemetry",
+    "name": "iglu_repo_telemetry",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "maxLength": 128,
+      "description": "The name of the repository"
+    },
+    "uri": {
+      "type": [
+        "string",
```

Comment:
Does local end up being `file://` or something for this?

Reply:
It isn't populated for local, that's why it's an optional field/accepts null. It's stored in localStorage, not a file so there's no URI for it.